### PR TITLE
Add some love to microbial variant calling tutorial

### PR DESCRIPTION
**Pull request by bebatut**
Fixing somethings spotted during Galaxy Africa training (ping @pvanheus):
- Use IUC tool
- Add some love to the tutorial:
  - Make the parameters fit to IUC tools
  - Split some hand-on sections
  - Add some question boxes
  - Add a bit more description about Snippy
  - Add links to download the data
- Add workflow

@Slugger70 Can you have a look if it is ok for you?

### CODE REVIEW

File: topics/variant-analysis/tutorials/microbial-variants/tutorial.md [hunk 1]
```diff
@@ -52,43 +52,51 @@ This data is available at Zenodo using the following [link](https://doi.org/10.5
...
+>     > * Paste the link into the text field
+>     > * Press **Start**
+>     {: .tip}
 >
 {: .hands_on}
 
-Your history pane should look like this after you have uploaded all of the data:
+# Find variants with Snippy
 
-![micro-start-history.png](../../images/micro-start-history.png)
```
Comment (Slugger70) [inline topics/variant-analysis/tutorials/microbial-variants/tutorial.md]: Is there a reason we get rid of this picture? We normally have them in place so the user can see that they are on the right track in the beginning for a bit of confidence etc.

Comment (pvanheus) [reply topics/variant-analysis/tutorials/microbial-variants/tutorial.md]: I agree with @Slugger70 - for users who are new to uploading to Galaxy it useful to show the starting point.

**Comment by pvanheus** [comment]
Thanks for the excellent updates @bebatut . Is there a need for a commentary on when to use JBrowse and when IGV might be better?

File: topics/variant-analysis/tutorials/microbial-variants/tutorial.md (hunk 1 above)
Comment (bgruening) [reply topics/variant-analysis/tutorials/microbial-variants/tutorial.md]: The rational behind removing images is that they are hard to update in general. Also please take into account that the "Galaxy introduction" is a pre-requirement to this tutorial, so I do not think it is so much useful at this point.

File: topics/variant-analysis/tutorials/microbial-variants/tutorial.md [hunk 2]
```diff
@@ -52,43 +52,51 @@ This data is available at Zenodo using the following [link](https://doi.org/10.5
...
+We will now run the Snippy tool on our reads, comparing it to the reference. 
 
-# Section 2: Find variants with Snippy
+Snippy is a rapid bacterial SNP calling and core genome alignments. Snippy finds SNPs between a haploid reference genome and your NGS sequence reads. It will find both substitutions (snps) and insertions/deletions (indels).
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/microbial-variants/tutorial.md]: s/is a rapid/is a tool for rapid/ ?

File: topics/variant-analysis/tutorials/microbial-variants/tutorial.md [hunk 3]
```diff
@@ -172,20 +192,17 @@ We could go through all of the variants like this and read them out of a text ta
...
->    A new file will be created in your history, this contains the JBrowse interactive visualisation. We will now view it's contents and play with it
->
-> 2. Inspect the `JBrowse on data XX and data XX - Complete` file by clicking on the eye icon
->
->    The JBrowse window will appear in the centre Galaxy panel.
->
-> 3. Display all the tracks and practice maneuvering around
+{: .hands_on}
+
+A new file will be created in your history, this contains the JBrowse interactive visualisation. We will now view it's contents and play with it by inspecting the `JBrowse on data XX and data XX - Complete` file (eye icon). The JBrowse window will appear in the centre Galaxy panel.
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/microbial-variants/tutorial.md]: s/it's/its/

**Comment by pvanheus** [comment]
Yes @bgruening I've been convinced by your reasoning.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**